dart/utils/urdf/DartLoader.cpp

```diff
@@ -374,8 +374,12 @@ dynamics::BodyNode* DartLoader::createDartJointAndNode(
   std::pair<dynamics::Joint*, dynamics::BodyNode*> pair;
   switch(_jt->type)
   {
-    case urdf::Joint::REVOLUTE:
     case urdf::Joint::CONTINUOUS:
+    {
+      singleDof.mPositionLowerLimits[0] = -math::constantsd::inf();
```

Comment:
Nit: Could we comment the reason of this overwriting something like:
```cpp
// We overwrite joint position limits to negative/positive infinities
// for "continuous" joint. This is because the URDF parser ignorantly
// reads joint limits or defaults to zero, which always should be
// infinity for "continuous" joint.
```